pyiron_base/jobs/flex/pythonfunctioncontainer.py

```diff
@@ -147,7 +147,14 @@ def run_static(self) -> None:
             with self._get_executor(max_workers=self.server.cores) as exe:
                 output = self._function(**input_dict, executor=exe)
         else:
-            output = self._function(**self.input.to_builtin())
+            if len(self.input.to_builtin()) > 0:
+                output = self._function(**self.input.to_builtin())
```

Comment:
```suggestion
                output = self._function(**self.input)
```
works even with empty data containers or functions that take no arguments, so there is no need to special case when you leave out the `to_builtin` call.

Reply:
I adopted your suggestion and removed the if-statement. I have to admit that I was not aware of the powerful implications we gain from deriving the `DataContainer` class from `MutableMapping`.